applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -10,7 +10,7 @@ menu "Bluetooth"
 
 config BLE_ACL_CONN_INTERVAL
 	int "BLE ACL Connection Interval (x*1.25ms)"
-	default 100
+	default 80
```

Comment:
This is to "make the controller more stable"?

Reply:
Yes, since 100ms ACL connection interval is more stable compared with 125ms.